corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/controller.js

```diff
@@ -105,7 +106,10 @@ hqDefine("cloudcare/js/formplayer/menus/controller", function () {
         if (menuResponse.breadcrumbs) {
             menusUtils.showBreadcrumbs(menuResponse.breadcrumbs);
             if (menuResponse.langs && menuResponse.langs.length > 1 && !appPreview && changeFormLanguage) {
-                menusUtils.showLanguageMenu(menuResponse.langs);
+                menusUtils.showLanguageMenu(menuResponse.langs, enablePrintOption);
+            }
+            else if (enablePrintOption) {
+                menusUtils.showPrintOption();
```

Comment:
This feels a little awkward that `showLanguageMenu` is sort-of separate from `showPrintOption`, but the former also needs to consider the state of the latter.  Have you considered switching to a single function that does both things (or either, or neither), something like:
```javascript
menusUtils.showFormMenu(enableLanguageMenu, enablePrintOption)
```

Reply:
ah yes agreed: [54ddad3](https://github.com/dimagi/commcare-hq/pull/32641/commits/54ddad33d3909c501597dd3d2d6ca96a29ab0faa)